plugin-gradle/src/main/java/com/diffplug/gradle/spotless/JavaExtension.java

```diff
@@ -59,17 +63,23 @@ public void importOrderFile(Object importOrderFile) {
 		addStep(ImportOrderStep.createFromFile(getProject().file(importOrderFile)));
 	}
 
+	/** Use {@link #eclipse()} instead */
```

Comment:
That would not be JavaDoc compliant. The comment is more fore the plugin developer. The user gets the "Use 'eclipse([version]).configFile(<file>)' instead." warning at the end of the build. Sounds reasonable to you?